yarn.lock

```diff
@@ -815,94 +815,63 @@
     unique-filename "^1.1.1"
     which "^1.3.1"
 
-"@jupyter-widgets/base@^2 || ^3":
-  version "3.0.0"
-  resolved "https://registry.yarnpkg.com/@jupyter-widgets/base/-/base-3.0.0.tgz#3996d566cddb742d275b007e0713de1e17f55a44"
-  integrity sha512-un1ZTHALCwE/SAYk2gEaonYM1JoaFyhosN8a3y2bhl4N26yCB3dP1PqGHLsAFur6ZB7fwuWwBEkIZx+nOwstAQ==
+"@jupyter-widgets/base@^2":
```

Comment:
Why is this changed without a corresponding change in package.json?

Reply:
I don't know, but `yarn bootstrap` also resolves this to `2.0.2` for me now.

Reply:
I think this is my fault. Somehow in https://github.com/visgl/deck.gl/pull/4573 I moved `@jupyter-widgets/base` to `peerDependencies` but either forgot to remove it from `dependencies` or rebased incorrectly or something. 

So I think I had previously built and pushed the `yarn.lock` having removed `@jupyter-widgets/base` from `dependencies`, and it resolved to `3.0.0`.